modules/33acrossBidAdapter.js

```diff
@@ -542,21 +599,21 @@ function interpretResponse(serverResponse, bidRequest) {
 // All this assumes that only one bid is ever returned by ttx
 function _createBidResponse(response) {
   const bid = {
-    requestId: response.id,
+    requestId: response.impid,
```

Comment:
just a question, if there are multiple seatbids and each seatbid has the same winning imp, then prebidjs will get two bidResponse obj with the same `requestId`, how will they handle this? just pick the highest price bid (i dont think the targeting are much different in this case because the bid response object does not containseat id)??

Reply:
hmm 33XCH will always do first price auction for imp therefore you'd get only the winning bid per imp which may not be  in the same seat as others, true. [The `requestId` in the object sent to Prebid corresponds to the Ad Unit ID](https://docs.prebid.org/dev-docs/bidder-adaptor.html#interpreting-the-response) (aka `bidId`). Very confusing I know.

Reply:
i think there won't be a second seat in the closest future, fingercross

Reply:
Yea assuming it'll all be one seat, but keeping this just in case